CMake/resolve_dependency_modules/README.md

```diff
@@ -31,7 +31,7 @@ by Velox. See details on bundling below.
 | xsimd             | 10.0.0          | Yes      |
 | re2               | 2021-04-01      | Yes      |
 | fmt               | 10.1.1          | Yes      |
-| simdjson          | 3.2.0           | Yes      |
+| simdjson          | 3.7.0           | Yes      |
```

Comment:
Do we actually REQUIRE this version? Or is it just a nice to have so we bump the version in resolve_dependency?

Reply:
Hi @assignUser, thanks for your review!
I posted some background message in PR description. As explained, we would like to use a simdjson API which is available since 3.7.0.

Reply:
Ok, then this change makes sense. Just wanted to  make sure we are not raising the minimum version needlessly. :+1: